### ISSUE TITLE
Underlying equations

### ISSUE THREAD

**Issue by ellisonbg** [UW-Madison-DataScience/Paltiel-COVID-19-Screening-for-College#3]
Hi, I am working to implement this model in python, and have found that the original equations in the article:

https://jamanetwork.com/journals/jamanetworkopen/fullarticle/2768923?utm_campaign=articlepdf&utm_medium=articlepdflink&utm_source=articlepdf&utm_content=jamanetworkopen.2020.16818

have some typos. Have you found and fixed those typos. Can you summarize them (will try to extract them from the code, but easier to look at the math).

**Comment by skent259** [comment]
We have had some correspondence with one of the authors, Dr. David Paltiel, who admitted the typos in the appendix of the JAMA paper. He seemed confident in his calculations from the [online spreadsheet](https://docs.google.com/spreadsheets/d/1otD4h-DpmAmh4dUAM4favTjbsly3t5z-OXOtFSbF1lY/edit#gid=1783644071), which our implementation copies.  So I believe that we have fixed these typos implicitly.

It might be easier to download his spreadsheet to an excel document, unhide the calculations tab, and look at those equations to build out in python.  That is the approach that we took in developing this tool, but unfortunately, it makes the code somewhat difficult to read.   

I do have a list of some typos to the paper from Dr. Paltiel, but I hesitate to copy them here.  Feel free to send me an email at skent259@gmail.com and I can share some additional information.

**Comment by aravamu2** [comment]
I believe that we have fixed these typos implicitly as well. Additionally, you can take a look at [calculations.Rmd](https://github.com/UW-Madison-DataScience/Paltiel-COVID-19-Screening-for-College/blob/master/calculations.Rmd) for a linear workflow of the implementation including calculation of the performance standards and generation of the stack area plot.

Let me know if you have any questions regarding the code. Thanks!

**Comment by ellisonbg** [comment]
I have been able to extract the equations from the R code and have verified they conserve the total population. Would be good to add a note somewhere on this repo about this, as I have a feeling this model is going to see a lot of usage in the coming weeks/months.

**Comment by aravamu2** [comment]
Will do. Thanks for the suggestion!
